### PR TITLE
Add documentation for WebSocket Endpoints

### PR DESCRIPTION
**Pull request by Thushani-Jayasekera**
https://github.com/wso2-enterprise/choreo/issues/30340

### CODE REVIEW

File: en/docs/testing/test-websocket-endpoints-via-the-websocket-console.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+# Test WebSocket Endpoints via the WebSocket Test Console
```
Comment (Nashaath) [inline en/docs/testing/test-websocket-endpoints-via-the-websocket-console.md:2]: Shall we go with "Test WebSocket Endpoints via the WebSocket Console"

File: en/docs/testing/test-websocket-endpoints-via-the-websocket-console.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Choreo provides an integrated WebSocket Console to test publicly exposed WebSocket endpoints of Service components you create and deploy. The WebSocket test Console allows you to connect to the service and interactively pass messages. As Choreo uses OAuth2.0 authentication by default to secure WebSocket APIs, the GraphQL Console generates test keys to test APIs.
+
+Follow the steps below to test a WebSocket endpoint via the WebSocket Console:
+
+1. Sign in to the [Choreo Console](https://console.choreo.dev/).
+
+2. In the **Component Listing** pane, click on the component you want to test.
+
+3. Click **Test** in the left navigation menu and then click **Console**. This opens the **GraphQL Console** pane.
```
Comment (Nashaath) [inline en/docs/testing/test-websocket-endpoints-via-the-websocket-console.md:12]: We need to replace all occurrences of the term "GraphQL" with "WebSocket" right?

**Review by Nashaath — APPROVED** [review]

**Comment by Thushani-Jayasekera** [comment]
Please merge https://github.com/wso2/docs-choreo-dev/pull/1615 after merging this PR.

File: en/docs/develop-components/develop-services/expose-a-websocket-endpoint-via-a-service.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+7. Under **Connect Your Repository**, enter the following information:
+
+    | **Field**               | **Value**               |
+    |-------------------------|-------------------------|
+    | **Organization**        | Your GitHub account     |
+    | **GitHub Repository**   | **`choreo-samples`**    |
+    | **Branch**              | **`main`**              |
+
+8. Select **Ballerina** as the buildpack.
```
Comment (Nashaath) [inline en/docs/develop-components/develop-services/expose-a-websocket-endpoint-via-a-service.md:78]: Should be **NodeJS** right?

File: en/site/references/choreo-limitations/index.html
Comment (Nashaath) [inline en/site/references/choreo-limitations/index.html:1]: This change is not necessary. The site directory content was added for testing purposes.

**Review by Nashaath — APPROVED** [review]

**Pull request merged**